gcp/PERMISSIONS.md

```diff
@@ -0,0 +1,97 @@
+This document is meant to be a description of the accounts that can manage a GCP cluster of the GPII at some level. Each account has a limited set of particular permissions which has been assigned in order to perform the cluster tasks.
+
+Accounts and permissions
+========================
+
+### cloud-admin@raisingthefloor.org
+
+  This is a Google group of which all the operators are members. The members of this group only have the permissions needed to allow IAM changes in the projects of the organization.
+
+  1. Organization Administrator
+
+### operator@raisingthefloor.org
+
+  By default an operator doesn't have permissions to manage any resources of the organization except the IAMs. In the case of an operator needs to escalate his/her personal privileges it will be possible such grant but only for a limited period of time.
+
+  Inherited permissions from _cloud-admin@raisingthefloor.org_ and can attach IAM roles just in special cases, and only for a limited period of time:
+
+  1. Billing Account Administrator
+  1. Support Account Administrator
+  1. Organization Policy Administrator
+  1. Owner
+
+### dev-user@raisingthefloor.org
+
```

Comment:
Same, clarify what is meant by `dev-user@raisingthefloor.org`, actual account or class of accounts?